app/views/AnalyticalFramework/components/WidgetEditor/DateWidgetForm/index.tsx

```diff
@@ -152,6 +152,9 @@ function DateWidgetForm(props: DateWidgetFormProps) {
             <Container
                 className={styles.container}
                 heading={value.title ?? 'Unnamed'}
+                headingDescription="(date)"
+                headingDescriptionClassName={styles.headingDescription}
+                spacing="compact"
```

Comment:
Instead of this, let's add the styling in the heading container.
You can use `headingContainerClassName` props for this.